source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -83,6 +109,35 @@ public ActionResult Calification(string period, string username)
             //2.c. Evaluator with evaluation done, show evaluation done
             //2.d. Responsible with evaluation done, show company evaluation to complete
             //2.e. Responsible with company evaluation done, show every evaluation including auto
+
+            //var evId = EmployeeEvaluation.GenerateEvaluationId(period, username);
+            var loggedUser = DetectUser();
+
+            RavenQueryStatistics stats;
+            EmployeeToEvaluate_Search.Projection projection = RavenSession
+                .Query<EmployeeToEvaluate_Search.Projection, EmployeeToEvaluate_Search>()
+                .Statistics(out stats)
+                .Where(e => e.UserName == username && e.Period == period)
+                .Customize(x => x.WaitForNonStaleResultsAsOfLastWrite()).FirstOrDefault();
+
+            // The evaluation doesn't exist
+            if (projection == null)
+            {
+                return HttpNotFound();
+            }
+
+            // If it's not the user, it must validate that it's responsible or evaluator
+            if (loggedUser != username)
+            {
+                if (projection.ResponsibleId != loggedUser)
+                {
+                    if (!(projection.Evaluators != null && projection.Evaluators.Contains(loggedUser)))
+                    {
+                        return HttpNotFound();
+                    }
+                }
+            }
```

Comment:
&& projection.ResponsibleId != loggedUser
    && (projection.Evaluators == null || !projection.Evaluators.Contains(loggedUser)))

For these two, the result should be HttpForbidden, rather than NotFound.